tests/api_fastapi/execution_api/routes/test_variables.py

```diff
@@ -23,10 +23,21 @@
 
 from airflow.models.variable import Variable
 
+from tests_common.test_utils.db import clear_db_variables
+
 pytestmark = pytest.mark.db_test
 
 
-class TestGetVariable:
+class TestVariableEndpoint:
+    @pytest.fixture(autouse=True)
+    def setup_method(self) -> None:
+        clear_db_variables()
+
+    def teardown_method(self) -> None:
+        clear_db_variables()
+
+
+class TestGetVariable(TestVariableEndpoint):
```

Comment:
I don't think we need a base class for this -- these can be done like this:

```suggestion
@pytest.fixture(autouse=True)
def setup_method(self) -> None:
    clear_db_variables()
    yield
    clear_db_variables()


class TestGetVariable::
```

Reply:
Handled, it doesn't have a `self`